WebApp-your-API/B2C/README.md

```diff
@@ -0,0 +1,368 @@
+        - For **Admin consent description** type `Allows the app to access ToDoListService-aspnetcore as the signed-in user.`
+        - For **User consent display name** type `Access ToDoListService-aspnetcore`
+        - For **User consent description** type `Allow the application to access ToDoListService-aspnetcore on your behalf.`
+        - Keep **State** as **Enabled**
+        - Select the **Add scope** button on the bottom to save this scope.
+
+#### Configure the service app (ToDoListService-aspnetcore) to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `Service\appsettings.json` file.
+1. Find the key `Instance` and replace the value with your tenant name. For example, `https://fabrikam.b2clogin.com`
+1. Find the key `ClientId` and replace the existing value with the application ID (clientId) of the `ToDoListService-aspnetcore` application copied from the Azure portal.
+1. Find the key `Domain` and replace the existing value with your Azure AD tenant name.
+1. Find the key `SignUpSignInPolicyId` and replace with the name of the `Sign up and sign in` policy you created.
+1. Find the key `ResetPasswordPolicyId` and replace with the name of the `Password reset` policy you created.
+1. Find the key `EditProfilePolicyId` and replace with the name of the `Profile editing` policy you created.
+
+### Register the Web app (WebApp-calls-API-blazor-server)
+
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD B2C** service.
+1. Select the **App Registrations** blade on the left, then select **New registration**.
+1. In the **Register an application page** that appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `WebApp-calls-API-blazor-server`.
+   - Under **Supported account types**, select **Accounts in any identity provider or organizational directory (for authenticating users with user flows)**.
+   - In the **Redirect URI (optional)** section, select **Web** in the combo-box and enter the following redirect URI: `https://localhost:44365/`.
+     > Note that there are more than one redirect URIs used in this sample. You'll need to add them from the **Authentication** tab later after the app has been created successfully.
+1. Select **Register** to create the application.
+1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select **Authentication** in the menu.
+   - If you don't have a platform added, select **Add a platform** and select the **Web** option.
+   - In the **Redirect URIs** section, enter the following redirect URIs.
```

Comment:
need to check implicit flow id token as well I think

Reply:
Apparently not -cancel that.

Reply:
Not required, it works. 
Thanks.